src/kit/LogViewer/LogViewer.tsx

```diff
@@ -38,6 +38,7 @@ export interface Props {
   serverAddress: (path: string) => string;
   sortKey?: keyof Log;
   title?: React.ReactNode;
+  height?: number;
 }
```

Comment:
Should there be a way to disable the resize functionality in case we don't want it?